provider/datagen/src/transform/segmenter/mod.rs

```diff
@@ -339,9 +339,14 @@ impl SegmenterRuleProvider {
                                     properties_map[c as usize] = property_index
                                 }
                             }
+
                             continue;
                         }
 
+                        // TODO:
```

Comment:
Suggestion: Do we want to address this issue? If so, please file a issue and insert the issue number here. 

It seems reasonable to default use UAX29 rules for Katakana unless dictionary defines something better. (I wonder how ICU4C behaves on Katakana.)